torch_xla/csrc/view.h

```diff
@@ -22,6 +22,14 @@ struct SelectInfo {
   xla::int64 stride = 0;
 };
 
+struct AsStridedInfo {
```

Comment:
Do you like it better as a struct? It only has one field and we end up with a few lines of boilerplate because of it.

Reply:
Yeah, I think it is clearer to have the single cases fields isolated into their own structure, instead of a bunch of random fields in the `ViewInfo` structure with eventual comments.